src/main/java/com/productdock/rbclibraryrental/RentalApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.productdock.rbclibraryrental;
```

Comment:
Let's use main package like this: com.productdock.library.rental
Later we will adopt this convention in catalog service too